packages/langium/src/lsp/language-server.ts

```diff
@@ -71,6 +71,7 @@ export class DefaultLanguageServer implements LanguageServer {
         const hasCodeActionProvider = this.hasService(e => e.lsp.CodeActionProvider);
         const hasSemanticTokensProvider = this.hasService(e => e.lsp.SemanticTokenProvider);
         const commandNames = this.services.lsp.ExecuteCommandHandler?.commands;
+        const documentLinkProvider = this.services.lsp.DocumentLinkProvider;
```

Comment:
In order to be in line with other providers, would it make sense to also call the `hasService` method here?
```suggestion
        const hasDocumentLinkProvider = this.hasService(e => e.shared.lsp.DocumentLinkProvider);
```

Reply:
Makes sense